bootstrap/090_post_setup.sql

```diff
@@ -379,6 +379,28 @@ CREATE OR REPLACE TASK TASKS.WAREHOUSE_SCHEDULING
     as
     call INTERNAL.UPDATE_WAREHOUSE_SCHEDULES(NULL, NULL);
 
+CREATE OR REPLACE TASK TASKS.SUNDECK_DEBUG
```

Comment:
Are we doing this as a task because that's the only way to retrieve the current organization?

I think I'm inclined to add to an existing task as opposed to create a new task.

Reply:
> Are we doing this as a task because that's the only way to retrieve the current organization?

Yeah, exactly.

> I think I'm inclined to add to an existing task as opposed to create a new task.

Sure, can do that, instead.